.github/CODEOWNERS

```diff
@@ -17,3 +17,4 @@
 /.github/CODEOWNERS          @garbas
 
 /full                        @tomberek
+/dotnet                      @Smaug123
```

Comment:
(Note that since I don't have write access to the repo, this CODEOWNERS line is actually useless, but it's what was in the instructions.)